### PR TITLE
Add generating version in middleman_protocol setup.py

### PR DESCRIPTION
**Pull request by pawelkisielewicz**
Resolves https://github.com/golemfactory/concent/issues/800

### CODE REVIEW

File: middleman_protocol/RELEASE-VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+0.8.0+dev194.gce34fd0
```
Comment (cameel) [inline middleman_protocol/RELEASE-VERSION]: We do not want to commit this file. Please remove it and add it to `.gitignore`.

In development you don't need it because `setup.py` can still import `version`.
In deployment we are going to generate it during the build.

File: middleman_protocol/setup.py [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+        return open(version_file, "r").read()
+    else:
+        from version import get_version
+        return get_version(prefix='v')
 
 
 setup(
     name='Middleman-Protocol',
-    version=get_version(prefix='v'),
+    version=generate_version(),
```
Comment (cameel) [inline middleman_protocol/setup.py]: This pull request changes only `middleman_protocol`. `signing_service` does this too so it also needs a fix (unless you're just planning to do it in a separate pull request).

Comment (pawelkisielewicz) [reply middleman_protocol/setup.py]: I will do it in this pull request. Done

**Comment by pawelkisielewicz** [comment]
I checked it before I created this pull request. It works
